### PR TITLE
Add Users.md

### PR DESCRIPTION
**Pull request by ineffectualproperty**
Signed-off-by: Kelly Olson <ineffectualproperty@users.noreply.github.com>

### CODE REVIEW

File: Users.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Use Case: Identity: Role-based Access Control  
+Project Link: https://opensource.t-mobile.com/blog/posts/hyperdirectory-blockchain-innovation/  
+Code: https://github.com/hyperledger/sawtooth-next-directory  
+Sawtooth Quote: “As the Un-carrier, T-Mobile approaches the wireless industry differently, from the customer experience to pushing the limits of the technologies that power our business and service,” said Warren McNeel, Senior Vice President, Digital Technology and Development, T-Mobile. “Using Hyperledger Sawtooth as a platform, the company created Sawtooth Hyper Directory as an Identity and Access Management (IAM) solution, and we’re excited to continue to develop new solutions on the platform that can help add value and functionality for both customers and our business.”  
+Contact:  
+
+**Tel Aviv Stock Exchange**  
+Use Case: Securities Lending  
+Project Link: https://www.ledgerinsights.com/tel-aviv-stock-exchange-blockchain/  
+Code:
```
Comment (peterschwarz) [inline Users.md:11]: Is the intention for these particular users to fill in these blanks?

Comment (ineffectualproperty) [reply Users.md:11]: Yep! Once we get this merged I'm going to reach out to the contacts I know to have them update it.

**Review by dcmiddle — APPROVED** [review]

**Review by vaporos — CHANGES_REQUESTED** [review]
This should probably be merged in a state ready for publishing.  Current thought is that we implement jekyll to facilitate this, probably with docker for testing purposes. Also, we may want to avoid the blank fields and add them when the information is added.

Lines need to be wrapped <80 chars.

Upper case character in the filename seems awkward from a URL perspective.

**Comment by vaporos** [comment]
This was integrated into the site, but the commit id changed so this didn't automatically close. Since it made it in, closing.

**Pull request closed without merging**